connector/servicegraphconnector/README.md

```diff
@@ -136,8 +136,8 @@ The following settings can be optionally configured:
   - Default: `1m`
 - `store_expiration_loop`: the time to expire old entries from the store periodically.
   - Default: `2s`
-- `virtual_node_peer_attributes`: the list of attributes need to match for building virtual server node, the higher the front, the higher the priority.
-  - Default: `[db.name, net.sock.peer.addr, net.peer.name, rpc.service, net.sock.peer.name, net.peer.name, http.url, http.target]`
+- `virtual_node_peer_attributes`: the list of attributes, ordered by priority, whose presence in a client span will result in the creation of a virtual server node.
```

Comment:
Since we will enable it by default, we also need to document how to disable it, it's a high-cardinality option.